src/data/director/navigation.tsx

```diff
@@ -1,6 +1,31 @@
 export const links = [
   {
-    name: "Interviews",
-    link: "/admin/interviews",
+    name: "Director",
+    link: () => "",
+  },
+  {
+    name: "Applications",
+    link: (program: string) => `/director/${program}/application`,
```

Comment:
prolly want all the functions to be the same parameter wise so it gives less problems